libs/localization/src/lib/en.json

```diff
@@ -1746,6 +1746,8 @@
         "macro": "Macro",
         "micro": "Micro"
       },
+      "classSelectionDropdown": "Select class(es)",
+      "iouThresholdDropdown": "IoU Threshold",
```

Comment:
Based on today's conversation, do we want to have a separate PR for iouThresholdDropdown? 

If it has a lower priority than classSelectionDropdown, maybe  iouThresholdDropdown can be worked on later on.